hack/pseudo-version/pseudo-version.go

```diff
@@ -91,9 +94,11 @@ func main() {
 		fmt.Println(headTime.Format(*timestampFormat))
 	case *printBranch:
 		fmt.Println(parsedBranch)
-	case *printReleaseVersion:
-		fmt.Println(strings.TrimPrefix(rawBranch, "release/"))
 	default:
-		fmt.Println(version)
+		if !strings.Contains(*base, "pre") {
+			fmt.Println(*base) // v2.7.0
+		} else {
+			fmt.Println(version) // 2.7.0-pre.0.20230313121936-bab76e8a9acf
+		}
```

Comment:
not sure i like the example but it does kind of help to understand what will be printed so feel free to keep it.

Reply:
I think I know what you mean. Let me add a bit more verbosity here.

Reply:
Thanks